presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotProjectExpressionConverter.java

```diff
@@ -78,8 +101,29 @@ protected boolean isImplicitCast(Type inputType, Type resultType)
         }
         return resultType.getTypeSignature().getBase().equals(StandardTypes.TIMESTAMP) && TIME_EQUIVALENT_TYPES.contains(inputType.getTypeSignature().getBase());
     }
-
-    private PinotExpression handleCast(
+    protected PinotExpression handleArithmeticExpression(
+            CallExpression expression,
+            OperatorType operatorType,
+            Map<VariableReferenceExpression, PinotQueryGeneratorContext.Selection> context)
+    {
+        List<RowExpression> arguments = expression.getArguments();
+        if (arguments.size() == 1) {
+            String prefix = operatorType == OperatorType.NEGATION ? "-" : "";
+            return derived(prefix + arguments.get(0).accept(this, context).getDefinition());
```

Comment:
can you give some examples for these negation cases?
Will the negation sign just work or the NOT statement is required?